src/k8s/requests/v2v/startV2VvmwareController.js

```diff
@@ -35,22 +35,36 @@ export const startV2VVMWareController = async ({ namespace }, { k8sGet, k8sCreat
     throw new Error('V2V VMWare: namespace must be selected');
   }
 
+  const name = V2VVMWARE_DEPLOYMENT_NAME;
```

Comment:
isnt this unnecessary ? why not use `V2VVMWARE_DEPLOYMENT_NAME` directly, instead of `name` ?

Reply:
good point, I can change

Reply:
The name is used in multiple places - I would rather leave it like it is